tree/traverse_test.go

```diff
@@ -0,0 +1 @@
+package tree_test
```

Comment:
?

Reply:
Lol. Yeah, used to contain tests and will contain tests in the future. Currently doing a gradual overhaul of which this is the first PR. Want me to remove for now?

Reply:
If you're aware of it and are planning to remove in a follow up, then its fine.